lib/kanta_web/live/translations/translation_form_live/components/plural_translation_form/plural_translation_form.html.heex

```diff
@@ -22,51 +22,59 @@
                 <svg class="flex-shrink-0 h-5 w-5 text-slate-500" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                   <path fill-rule="evenodd" d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" clip-rule="evenodd"></path>
                 </svg>
-                <a phx-click="navigate" phx-value-to={"/kanta/locales/#{@locale.id}/translations"} class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= @locale.name %></a>
+                <.link patch={"/kanta/locales/#{@locale.id}/translations"} class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= @locale.native_name %></.link>
               </div>
             </li>
             <li>
               <div class="flex items-center">
                 <svg class="flex-shrink-0 h-5 w-5 text-slate-500" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                   <path fill-rule="evenodd" d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" clip-rule="evenodd"></path>
                 </svg>
-                <a href="#" aria-current="page" class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= @message.msgid %></a>
+                <a href="#" aria-current="page" class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= String.slice(@message.msgid, 0..30) %></a>
```

Comment:
Is this `String.slice/2` to prevent overflow?
I think a css solution should be preferred:

https://tailwindcss.com/docs/text-overflow#truncate